0.3/schemas/json_schema/image.schema

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "http://localhost:8000/image.schema",
+  "$id": "https://ngff.openmicroscopy.org/0.3/schemas/json_schema/image.schema",
```

Comment:
The "schema(s)" in this URL are a bit much....

Reply:
That is just the path we currently have in this repo. Since we're only supporting `json_schema` now, we could drop that dir. But the paths are even more painful at https://github.com/ome/ome-zarr-py/pull/142 where I wanted to maintain the same directory structure as here, simply replacing `https://ngff.openmicroscopy.org/` with a single path in that repo (expecting that we'll package the schemas as part of the release). So the path in that repo then becomes `schemas/0.3/schemas/json_schema/image.schema`! 
The problem with is that in the NGFF repo, the version is at the top level, so we have a `schemas` directory under each version, but it's the other way around on `ome-zarr-py` where I wanted all the schemas to be under a top-level `schemas/`.
Need to unify the approach between the repos. Do we create top-level version dirs on `ome-zarr-py`?